src/providers/openstack/configdrive.rs

```diff
@@ -228,6 +233,10 @@ mod tests {
             "abai-fcos-afterburn-test"
         );
         assert_eq!(parsed.availability_zone.unwrap_or_default(), "nova");
+        assert_eq!(
+            parsed.uuid.unwrap_or_default(),
+            "b3c7f4de-da0b-44ae-a42c-fa3806b61d7f"
```

Comment:
Novice question, whats this uuid? is it standard or random default?

Reply:
It's the value that was already there [in the fixture](https://github.com/shiftstack/afterburn/blob/6418459606b0c6e2664a88d5a57552727c34c5d4/tests/fixtures/openstack-config-drive/openstack/meta_data.json#L4)